Yoast/wordpress-seo
# P1-133 Merge robot meta tags

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Merges extra googlebot and bingbot meta tag values into the robots meta tag value. Deprecating the `Googlebot_Presenter` and `Bingbot_Presenter`.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* For posts, pages and custom posts, check the robot output on the frontend:
  * No longer output `googlebot` and `bingbot` meta tags.
  * Robots meta tag includes `max-snippet:-1, max-image-preview:large, max-video-preview:-1`.
* Edit the "indexable" and change the `Meta robots advanced` setting:
  * Adding `No Image Index` should remove `max-image-preview:large` and add `noimageindex`.
  * Adding `No Archive` should add `noarchive`.
  * Adding `No Snippet` should remove `max-snippet:-1` and add `nosnippet`.
* Enabling/disabling the index should change the robots value to have `noindex` and not output any of the extra tags (except the follow/nofollow).
* Enabling/disabling the follow should change the robots value to have `nofollow`.

## UI changes

## Documentation

## Quality assurance


Fixes https://yoast.atlassian.net/browse/P1-133
